src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -271,6 +287,7 @@ export type CloudWatchLogsParameters = {
     endTime?: number
     limit?: number
     streamName?: string
+    streamNameOptions?: string[]
```

Comment:
In almost every way they are the same thing. The `streamNameOptions` is for limiting a log group to certain log streams whereas the `streamName` is for describing what stream is being viewed. I decided to keep them separate because this would allow for the functionality of limiting a log group search to include contents from multiple different streams (which the API supports). It also allows me reserve `streamName` to imply that we are viewing a single stream in `isLogStream`.